<?xml version='1.0' encoding='UTF-8' standalone='no'?>
<doxygen xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="compound.xsd" version="1.8.11">
  <compounddef id="classcomposite__manager__client_1_1exceptions_1_1ApiValueError" kind="class" language="Python" prot="public">
    <compoundname>composite_manager_client::exceptions::ApiValueError</compoundname>
    <basecompoundref refid="classcomposite__manager__client_1_1exceptions_1_1OpenApiException" prot="public" virt="non-virtual">composite_manager_client.exceptions.OpenApiException</basecompoundref>
    <basecompoundref prot="public" virt="non-virtual">ValueError</basecompoundref>
    <basecompoundref refid="classcomposite__manager__client_1_1exceptions_1_1OpenApiException" prot="public" virt="non-virtual">composite_manager_client.exceptions.OpenApiException</basecompoundref>
    <basecompoundref prot="public" virt="non-virtual">ValueError</basecompoundref>
      <sectiondef kind="public-attrib">
      <memberdef kind="variable" id="classcomposite__manager__client_1_1exceptions_1_1ApiValueError_1ac0db6667c4cc44a966a9212155cfa70d" prot="public" static="no" mutable="no">
        <type></type>
        <definition>composite_manager_client.exceptions.ApiValueError::path_to_item</definition>
        <argsstring></argsstring>
        <name>path_to_item</name>
        <briefdescription>
        </briefdescription>
        <detaileddescription>
        </detaileddescription>
        <inbodydescription>
        </inbodydescription>
        <location file="OpenAPI/tests/composite_manager_client/composite_manager_client/exceptions.py" line="60" column="1" bodyfile="OpenAPI/tests/composite_manager_client/composite_manager_client/exceptions.py" bodystart="60" bodyend="-1"/>
      </memberdef>
      </sectiondef>
      <sectiondef kind="public-func">
      <memberdef kind="function" id="classcomposite__manager__client_1_1exceptions_1_1ApiValueError_1ab53643ae18191aa9aad8cef3a02e99fb" prot="public" static="no" const="no" explicit="no" inline="no" virt="non-virtual">
        <type>def</type>
        <definition>def composite_manager_client.exceptions.ApiValueError.__init__</definition>
        <argsstring>(self, msg, path_to_item=None)</argsstring>
        <name>__init__</name>
        <param>
          <type>self</type>
          <defname>self</defname>
        </param>
        <param>
          <type>msg</type>
          <defname>msg</defname>
        </param>
        <param>
          <type>path_to_item</type>
          <defname>path_to_item</defname>
          <defval>None</defval>
        </param>
        <briefdescription>
        </briefdescription>
        <detaileddescription>
<para><verbatim>Args:
    msg (str): the exception message

Keyword Args:
    path_to_item (list) the path to the exception in the
received_data dict. None if unset
</verbatim> </para>        </detaileddescription>
        <inbodydescription>
        </inbodydescription>
        <location file="OpenAPI/tests/composite_manager_client/composite_manager_client/exceptions.py" line="50" column="1" bodyfile="OpenAPI/tests/composite_manager_client/composite_manager_client/exceptions.py" bodystart="50" bodyend="66"/>
      </memberdef>
      <memberdef kind="function" id="classcomposite__manager__client_1_1exceptions_1_1ApiValueError_1ab53643ae18191aa9aad8cef3a02e99fb" prot="public" static="no" const="no" explicit="no" inline="no" virt="non-virtual">
        <type>def</type>
        <definition>def composite_manager_client.exceptions.ApiValueError.__init__</definition>
        <argsstring>(self, msg, path_to_item=None)</argsstring>
        <name>__init__</name>
        <param>
          <type>self</type>
          <defname>self</defname>
        </param>
        <param>
          <type>msg</type>
          <defname>msg</defname>
        </param>
        <param>
          <type>path_to_item</type>
          <defname>path_to_item</defname>
          <defval>None</defval>
        </param>
        <briefdescription>
        </briefdescription>
        <detaileddescription>
<para><verbatim>Args:
    msg (str): the exception message

Keyword Args:
    path_to_item (list) the path to the exception in the
received_data dict. None if unset
</verbatim> </para>        </detaileddescription>
        <inbodydescription>
        </inbodydescription>
        <location file="OpenAPI/tests/editable_copies/composite_manager_client/composite_manager_client/exceptions.py" line="50" column="1" bodyfile="OpenAPI/tests/editable_copies/composite_manager_client/composite_manager_client/exceptions.py" bodystart="50" bodyend="66"/>
      </memberdef>
      </sectiondef>
    <briefdescription>
    </briefdescription>
    <detaileddescription>
    </detaileddescription>
    <inheritancegraph>
      <node id="2055">
        <label>composite_manager_client.exceptions.ApiValueError</label>
        <link refid="classcomposite__manager__client_1_1exceptions_1_1ApiValueError"/>
        <childnode refid="2056" relation="public-inheritance">
        </childnode>
        <childnode refid="2058" relation="public-inheritance">
        </childnode>
        <childnode refid="2056" relation="public-inheritance">
        </childnode>
        <childnode refid="2058" relation="public-inheritance">
        </childnode>
      </node>
      <node id="2058">
        <label>ValueError</label>
      </node>
      <node id="2057">
        <label>Exception</label>
      </node>
      <node id="2056">
        <label>composite_manager_client.exceptions.OpenApiException</label>
        <link refid="classcomposite__manager__client_1_1exceptions_1_1OpenApiException"/>
        <childnode refid="2057" relation="public-inheritance">
        </childnode>
        <childnode refid="2057" relation="public-inheritance">
        </childnode>
      </node>
    </inheritancegraph>
    <collaborationgraph>
      <node id="2059">
        <label>composite_manager_client.exceptions.ApiValueError</label>
        <link refid="classcomposite__manager__client_1_1exceptions_1_1ApiValueError"/>
        <childnode refid="2060" relation="public-inheritance">
        </childnode>
        <childnode refid="2062" relation="public-inheritance">
        </childnode>
        <childnode refid="2060" relation="public-inheritance">
        </childnode>
        <childnode refid="2062" relation="public-inheritance">
        </childnode>
      </node>
      <node id="2062">
        <label>ValueError</label>
      </node>
      <node id="2061">
        <label>Exception</label>
      </node>
      <node id="2060">
        <label>composite_manager_client.exceptions.OpenApiException</label>
        <link refid="classcomposite__manager__client_1_1exceptions_1_1OpenApiException"/>
        <childnode refid="2061" relation="public-inheritance">
        </childnode>
        <childnode refid="2061" relation="public-inheritance">
        </childnode>
      </node>
    </collaborationgraph>
    <location file="OpenAPI/tests/composite_manager_client/composite_manager_client/exceptions.py" line="49" column="1" bodyfile="OpenAPI/tests/composite_manager_client/composite_manager_client/exceptions.py" bodystart="49" bodyend="67"/>
    <listofallmembers>
      <member refid="classcomposite__manager__client_1_1exceptions_1_1ApiValueError_1ab53643ae18191aa9aad8cef3a02e99fb" prot="public" virt="non-virtual"><scope>composite_manager_client::exceptions::ApiValueError</scope><name>__init__</name></member>
      <member refid="classcomposite__manager__client_1_1exceptions_1_1ApiValueError_1ab53643ae18191aa9aad8cef3a02e99fb" prot="public" virt="non-virtual"><scope>composite_manager_client::exceptions::ApiValueError</scope><name>__init__</name></member>
      <member refid="classcomposite__manager__client_1_1exceptions_1_1ApiValueError_1ac0db6667c4cc44a966a9212155cfa70d" prot="public" virt="non-virtual"><scope>composite_manager_client::exceptions::ApiValueError</scope><name>path_to_item</name></member>
    </listofallmembers>
  </compounddef>
</doxygen>
